check-api-docs.bash

```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/env bash
 set -e
 
-PROJECT=${1?No project specified}
+JSON_TASK=${1?No json task specified}
+WORKING_DIR=${2:-.}
```

Comment:
change needed for prisoner search, but might as well copy here too (defaulting to `.` for current directory)